From 35275512f83d246ddbddd2be2b970925e73df45e Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Mon, 9 Oct 2017 14:25:26 +0200
Subject: [PATCH] doc: Don't document internal endian macros

As those will change across platforms, and lead to multilib conflicts
---
 doc/Doxyfile.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/Doxyfile.in b/doc/Doxyfile.in
index 65c2b83..1bc3b16 100644
--- a/doc/Doxyfile.in
+++ b/doc/Doxyfile.in
@@ -452,7 +452,7 @@ EXCLUDE_SYMLINKS       = NO
 # EXCLUDE_PATTERNS tag to specify one or more wildcard patterns to exclude 
 # certain files from those directories.
 
-EXCLUDE_PATTERNS       = 
+EXCLUDE_PATTERNS       = */src/gphoto2-endian.h
 
 # The EXAMPLE_PATH tag can be used to specify one or more files or 
 # directories that contain example code fragments that are included (see 
-- 
2.14.2

